doc/help.md

```
@@ -0,0 +1,5 @@
+## Test dependencies:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)